requirements-dev.in

```diff
@@ -1,6 +1,7 @@
 # Requirements for installing other packages
 pip
 setuptools>=18.5
+pip-tools==6.5.0
```

Comment:
pip-tools 6.5.0 fixes the issue that prevents it from working with pip 22, so we should at least require 6.5.0.